docs/sources/alerting/alert-groups/view-alert-grouping.md

```diff
@@ -9,13 +9,13 @@ keywords:
   - alerting
   - alerts
   - groups
-title: View alert groupings
+title: View alert groups
 weight: 445
 ---
 
-# View alert groupings
+# View alert groups
 
-To view alert groupings:
+To view alert groups:
 
 1. In the Grafana menu, click the **Alerting** (bell) icon to open the Alerting page listing existing alerts.
 1. Click **Alert grouping** to open the page listing existing groups.
```

Comment:
```suggestion
1. Click **Alert groups** to open the page listing existing groups.
```
Important one down here

(and discussed in Slack: it makes sense to hold off changing the filename / links, for now)

Reply:
Corrected :)